cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -697,7 +697,7 @@ def compose_color(self,component_name,parameter_name):
 			full_parameter_name = self.ns_global_prefix + "/" + component_name + "/" + parameter_name
 			if not rospy.has_param(full_parameter_name):
 				rospy.logerr("parameter %s does not exist on ROS Parameter Server, aborting...",full_parameter_name)
-				return 2
+				return 2, None
```

Comment:
can you elaborate on what exactly the two values mean?
Is it error code and handle?

Is it always the case that a tuple should be returned?
It doesn't seem to be done consistently within the simple_script_server....
Could you review the this file in order to see whether the same fix should be applied for other functions, too?